inventory.go

```diff
@@ -29,99 +29,75 @@ func main() {
 	props := getPropsFromConfig()
 	args := CLIArgs{}
 	arg.MustParse(&args)
-	// If argument "--list" was set to true
 	if args.List {
 		output, err := handleList(props)
 		if err != nil {
 			panic(fmt.Errorf("Fatal error handling list: %s \n", err))
 		}
 		marshalResult, _ := json.MarshalIndent(output, "", "  ")
 		fmt.Println(string(marshalResult))
-		// If argument "--host" was set instead of host.
-	} else if args.Host != "" {
-		output, err := handleHost(args.Host, props)
-		if err != nil {
-			panic(fmt.Errorf("Fatal error handling host: %s \n", err))
-		}
-		fmt.Printf("%+v\n", output)
+
+		return
 	}
 }
 
 // getPropsFromConfig returns a props instance with values from config.yml
 func getPropsFromConfig() props {
-	// config allows us to set the path for the configuration yaml file and retrieve its values
 	config := viper.New()
 
-	// envRackhdApiUrl is the RackHD REST API URL defined by an environment variable and
 	envRackhdApiUrl, envRackhdApiUrlOk := os.LookupEnv(RackHdApiUrlEnvVarName)
 	envAnsibleRackhdConfigPath, envAnsibleRackhdConfigPathOk := os.LookupEnv(AnsibleRackHdConfigPath)
-	// If the "ANSIBLE_RACKHD_CONFIG_PATH" variable is set we define that as our config file
 	if envAnsibleRackhdConfigPathOk == true {
 		config.SetConfigFile(envAnsibleRackhdConfigPath)
-		// Else we define the config.yaml as the default config file
 	} else {
 		config.AddConfigPath(".")
 		config.SetConfigName("config")
 		config.SetConfigType("yml")
 	}
-	// ReadConfig reads our config file
 	err := config.ReadInConfig()
 	if err != nil {
 		panic(fmt.Errorf("Fatal error config file: %s \n", err))
 	}
 
-	// If the "RACK_HD_API_URL" is set we define that as our URL API
 	var rackhdUrl string
 	if envRackhdApiUrlOk == true {
 		rackhdUrl = envRackhdApiUrl
 	} else {
 		rackhdUrl = config.GetString("rackhd_api_url")
 	}
 
-	return props{rackhdUrl:rackhdUrl, groups: config.GetStringSlice("groups"), filterGroup: config.GetString("filter_group")}
+	return props{
+		rackhdUrl: rackhdUrl, groups: config.GetStringSlice("groups"),
```

Comment:
if you're splitting, do it on argument, KV p/ line

Reply:
I did, then ran "go fmt" and it must have changed that line.
I will pay attention to that next time.